store/postgres/src/relational_queries.rs

```diff
@@ -1523,43 +1594,58 @@ impl QueryFragment<Pg> for FilterRange {
 /// The parallel to `EntityQuery`.
 ///
 /// Details of how query generation for `FilterQuery` works can be found
-/// in `docs/implementation/query-prefetching.md`
+/// `https://github.com/graphprotocol/rfcs/blob/master/engineering-plans/0001-graphql-query-prefetching.md`
 #[derive(Debug, Clone)]
 pub struct FilterQuery<'a> {
-    collection: FilterCollection<'a>,
-    sort_key: SortKey,
+    collection: &'a FilterCollection<'a>,
```

Comment:
For some reason, I can't add to the discussion we had before about this - but why make this a reference in the first place? There shouldn't be an issue with having that owned by `FilterQuery`

Reply:
I'm remembering another reason I created the FilterCollection outside of FilterQuery::new(). I update SortKey to contain an entire Column instead of just Column.name, so now a Column reference is returned from FilterQuery::new() in the SortKey, this Column references the FilterCollection.